coil-gif/README.md

```diff
@@ -0,0 +1,27 @@
+# coil-gif
```

Comment:
Let's move the existing `docs/gifs.md` to `coil-gif/README.md` and we can copy it back during the deploy process.